examples/manager.py

```diff
@@ -0,0 +1,24 @@
+'''
+Waits for a keyboard interupt, and sends a kill signal to the other nodes
+'''
+
+import time
+import zmq
+
+
+def main():
+    context = zmq.Context()
+    shutdown = context.socket(zmq.PUB)
+    shutdown.bind("tcp://*:9000")
+
+    shutdown.send(b'START')
+    try:
+        while True:
+            time.sleep(1)
+    except KeyboardInterrupt:
+        print("Sending kill signal")
+        shutdown.send(b"EXIT")
```

Comment:
You may want to use `send_multipart([b'EXIT', reason])` so we can use `recv_multipart()` all the time.